sky/spot/constants.py

```diff
@@ -13,7 +13,7 @@
 # OOM (each vCPU can have 4 spot controller processes as we set the CPU
 # requirement to 0.25, and 3 GB is barely enough for 4 spot processes).
 # We use 50 GB disk size to reduce the cost.
-CONTROLLER_RESOURCES = {'memory': '3x', 'disk_size': 50}
+CONTROLLER_RESOURCES = {'cpus': '8+', 'memory': '3x', 'disk_size': 50}
```

Comment:
Update L9-11?

Reply:
Oh, I meant that those lines suggest concrete instance types, and here we do not necessarily use those? E.g., those VMs have 32GB, and the spec on this line suggests >= 24GB?

Reply:
Updated the comments a little bit :)